includes/class-sensei-teacher.php

```diff
@@ -935,6 +937,8 @@ public function notify_admin_teacher_course_creation( $new_status, $old_status,
 
         // Send mail
         Sensei()->emails->send( $recipient, $subject , Sensei()->emails->get_content( $template ) );
+				
+				do_action('sensei_after_sending_email');
```

Comment:
Could we move these actions into the send function of the email class? They should both have the same arguments. I would want to give the client  all the email parameters in order to make it useful.